pkgs/development/compilers/gambit/gambit-support.nix

```diff
@@ -19,21 +20,22 @@ rec {
         --replace "$(grep '^PACKAGE_VERSION=.*$' configure)" 'PACKAGE_VERSION="v${git-version}"' \
         --replace "$(grep '^PACKAGE_STRING=.*$' configure)" 'PACKAGE_STRING="Gambit v${git-version}"' ;
     '';
-    targets = "arm,java,js,php,python,riscv-32,riscv-64,ruby,x86,x86-64"; # eats 100% cpu on _digest
-    modules = false;
+    targets = "js"; # arm,java,js,php,python,riscv-32,riscv-64,ruby,x86,x86-64
```

Comment:
Why removal of arm/x86-64 native targets? Purely for build time reasons?

Reply:
There are various failures when building "modules" that don't have time to address. Also, while basically working, these backends are missing many features, and they are receiving to love at the moment. Hopefully at some point someone will make them really work, but until then, it's too much hassle for too little upside.